app/models/user.rb

```diff
@@ -131,4 +134,12 @@ def recently_reached_by_user?(preg)
   def recently_called_by_user?(preg)
     preg.calls.any? { |call| call.created_by_id == id && call.recent? }
   end
+
+  def needs_password_change_email?
+    encrypted_password_changed? && persisted?
+  end
+
+  def send_password_change_email
+    UserMailer.password_changed(id).deliver
```

Comment:
I saw the "deprecated" warnings and changed it in my current local copy. I've just been struggling with that user id part this whole time so I haven't pushed it.